content/05.main-text.md

```diff
@@ -111,81 +110,23 @@ or translational distance models [@transe].
-The workflow is based around showing the user the most important metapaths and paths for a pair of query nodes.
-
-The design guides the user through selecting a source and target node (Figure {@fig:webapp}A).
-The webapp returns metapaths, scored by whether they occurred more than expected based on network degree (Figure {@fig:webapp}B).
-Users can proceed by requesting the specific paths for each metapath,
-which are placed in a unified table sorted according to their path score (Figure {@fig:webapp}C).
-Finally, the webapp produces publication-ready visualizations containing user-selected paths (Figure {@fig:webapp}D).
-
-![
-**Using the connectivity search webapp to explore the pathophysiology of Alzheimer's disease.**
-This figure shows an example user workflow for <https://het.io/search/>.
-\
-**A.**
-The user selects two nodes.
-Here, the user is interested in Alzheimer's disease, so [selects this](https://het.io/search/?source=17287) as the source node.
-The user limits the target node search to metanodes relating to gene function.
-The target node search box suggests nodes, sorted by the number of significant metapaths.
-When the user types in the target node box, the matches reorder based on search word similarity.
-Here, the user becomes interested in how the circadian rhythm might relate to Alzheimer's disease.
-\
-**B.**
-The webapp returns metapaths between Alzheimer's disease and the circadian rhythm pathway.
-The user unchecks "precomputed only" to compute results for all metapaths with length ≤ 3, not just those that surpass the database inclusion threshold.
-The user sorts by adjusted _p_-value and [selects][alzheimer-metapaths] 7 of the top 10 metapaths.
-\
-**C.**
-Paths for the selected metapaths are ordered by their path score.
-The user selects 8 paths (1 from a subsequent page of results) to show in the graph visualization and highlights a single path involving _ARNT2_ for emphasis.
-\
-**D.**
-A subgraph displays the previously selected paths.
-The user improves on the automated layout by repositioning nodes.
-Clicking an edge displays its properties, informing the user that association between Creutzfeldt-Jakob disease and _NPAS2_ was detected by GWAS.
-<!--
-  This example is also used at https://slides.com/dhimmel/rocky2019#/4
-  More info at https://github.com/greenelab/connectivity-search-manuscript/issues/7
--->
-](https://github.com/greenelab/connectivity-search-manuscript/raw/b290b4ad435553c7126867e0720f3112b4692809/content/media/webapp/v3/webapp.png){#fig:webapp width="100%" .white}
-
-[alzheimer-metapaths]: https://het.io/search/?source=17287&target=7607&metapaths=DaGiGpPW%2CDdGiGpPW%2CDdGpPW%2CDlAeGpPW%2CDrDaGpPW%2CDrDuGpPW%2CDuGiGpPW&complete=
+Completing hetnet connectivity search involved advances on three fronts.
+We implemented new software for efficient software for matrix-based operations on hetnets.
+We developed strategies to efficiently calculate an appropriate metric under the null.
```

Comment:
Double check this sentence.

Reply:
Can you check the new sentence? I'm trying to avoid saying DWPC, but also to give an idea of the work there.